tools/dds/dds-sniffer/dds-sniffer.cpp

```diff
@@ -12,26 +13,31 @@
 #include <fastrtps/types/DynamicDataHelper.hpp>
 #include <fastrtps/types/DynamicDataFactory.h>
 
-#include "tclap/CmdLine.h"
-#include "tclap/ValueArg.h"
-#include "tclap/SwitchArg.h"
+#include <tclap/CmdLine.h>
+#include <tclap/ValueArg.h>
+#include <tclap/SwitchArg.h>
 
 using namespace TCLAP;
 
+//GUID_t - 4 MSB bytes host, 4 bytes process, 4 bytes participant, 4 bytes entity ID (reader/writer)
```

Comment:
I still think it's unclear.
Can you give an example of what a whole GUID looks like, and which part of it we want?